actor/v2action/service_access_test.go

```diff
@@ -495,6 +513,29 @@ var _ = Describe("Service Access", func() {
 				Expect(orgGUID2).To(Equal("org-guid-1"))
 				Expect([]string{planGUID1, planGUID2}).To(ConsistOf([]string{"service-plan-guid-1", "service-plan-guid-2"}))
 			})
+
+			When("service is already enabled for all orgs for a plan", func() {
+				BeforeEach(func() {
+					fakeCloudControllerClient.GetServicePlansReturns(
+						[]ccv2.ServicePlan{
+							{Name: "plan-1", GUID: "service-plan-guid-1", Public: true},
+							{Name: "plan-2", GUID: "service-plan-guid-2", Public: false},
+						},
+						nil,
```

Comment:
Please add test that any warnings returned from the API layer are properly handled.